deploy/cephfs/kubernetes/csi-nodeplugin-rbac.yaml

```diff
@@ -9,9 +9,6 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: cephfs-csi-nodeplugin
 rules:
-  - apiGroups: [""]
```

Comment:
Why keep empty ClusterRole without any rules?

Reply:
removed it